Model/Behavior/SerializableBehavior.php

```diff
@@ -91,7 +91,7 @@ public function beforeSave(Model $Model, $options = array()) {
  * @param array $options
  * @return void
  */
-	public function afterSave(Model$Model, $created, $options = array()) {
+	public function afterSave(Model $Model, $created, $options = array()) {
```

Comment:
PSR-2 Would have us move the opening bracket to a newline, but I suspect most of this plugin doesn't follow PSR-2.